staff/Melinx/blog/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog</title>    
+    <link rel="icon" href="images/basic-red-heart.png" type="image/png" sizes="16x16">
+
+</head>
+
+<body>
+    <main>
+        <h1>~ How to Make a Nicely Sturctured Blog ~</h1>
+
+        <h3>HINTS shared in Slack:</h3>
+        <ul>
+            <li>ONE! <a href="https://www.hongkiat.com/blog/html-5-semantics/" target="blank">A Look Into Proper HTML5 Semantics</a></li>
+            <li>TWO! <a href="https://webflow.com/blog/html5-semantic-elements-and-webflow-the-essential-guide"target="blank"> ...and Webflow: the essential guide</a></li>
+            
+        </ul>
+        <article>
+            The article element represents a complete, or self-contained, composition in a document, page, application, or site and that
```

Comment:
This text should contained inside a Tag (f. ex. in a `<p>`)
`<article>`
  `<p>bla bla bla...</p>`
`</article>`